### PR TITLE
Update to operator-sdk 0.9.0

### PR DESCRIPTION
**Pull request by sthaha**
This PR introduces the following changes
- upgrades to operator-sdk version 0.9.0 which makes use of go mod
- moves from dep to go-mod
- updates vendor/
- adapts to changes to `make build`

Signed-off-by: sthaha@redhat.com

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -25,8 +32,9 @@ import (
 
 // Change below variables to serve metrics on different host or port.
 var (
-	metricsHost       = "0.0.0.0"
-	metricsPort int32 = 8383
+	metricsHost               = "0.0.0.0"
+	metricsPort         int32 = 8383
+	operatorMetricsPort int32 = 8686
```
Comment (chmouel) [inline cmd/manager/main.go]: (Non related) Do we have plan to do something with those metrics port ?

Comment (sthaha) [reply cmd/manager/main.go]: part of `operator-sdk` `0.9.1`

Comment (sthaha) [reply cmd/manager/main.go]: we may, there is a plan to do operator metering.

**Pull request merged**